operator/src/main/java/org/keycloak/operator/crds/v2alpha1/deployment/spec/Truststore.java

```diff
@@ -0,0 +1,33 @@
+package org.keycloak.operator.crds.v2alpha1.deployment.spec;
```

Comment:
Could you please add the license?

Reply:
Added now. Just to double check on this. There doesn't seem to be consistency (it's missing in quite a few source files), or an automated check to enforce that it's included. I know we require one in fabric8, but it's a dumb check so we're forced to use the same copyright year for everything. It also appears in keycloak that the copyright year is set for each file, is that correct as well?

Reply:
Like, I don't know the license requirements for the Keycloak code, but AFAIK, it's been a general convention to put the license to new classes. But it's not a hard requirement, I guess. It just blicked to my eye :)